### PR TITLE
[NETSHELL] Add file hdr. Improve portuguese (pt-PT) translation.

### PR DESCRIPTION
**Pull request by zecarlos1957**
## Purpose

Add file hdr. 
Improve portuguese (pt-PT) translation.
Fix typos.

### CODE REVIEW

File: dll/shellext/netshell/lang/pt-PT.rc [hunk 1]
```diff
@@ -106,13 +113,13 @@ END
...
+CAPTION "Programa de configuração do ReactOS"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Pretende que este computador seja membro de um domínio?\n\
-(Pode obter esta informação do seu administrador de rede.)", IDC_STATIC, 33, 2, 263, 50
-    AUTORADIOBUTTON "Não, este computador fará parte de um grupo de &trabalho.", IDC_SELECT_WORKGROUP, 33, 32, 253, 20, WS_TABSTOP | WS_GROUP
-    AUTORADIOBUTTON "Sim, este computador fará parte de um &domínio.", IDC_SELECT_DOMAIN, 33, 47, 253, 20, WS_TABSTOP
+    LTEXT "Deseja que este computador seja membro de um domínio?\n\
+(Pode obter esta informação no seu administrador de rede.)", IDC_STATIC, 33, 2, 263, 50
+    AUTORADIOBUTTON "Não; fazer este computador membro do seguinte grupo de trabalho.", IDC_SELECT_WORKGROUP, 33, 32, 253, 20, WS_TABSTOP | WS_GROUP
```
Comment (JoachimHenze) [inline dll/shellext/netshell/lang/pt-PT.rc:121]: Why did you strip the accelerator? To be honest, I can't spot a single substantial improvement in this PR, except line 119 Pretende -> Deseja and maybe the compone**n**tes typo-fix.

**Comment by JoachimHenze** [comment]
Most of this PR is bloat. To be honest, I can't spot a single substantial improvement in this PR, except line 119 Pretende -> Deseja and maybe the componentes typo-fix.

File: dll/shellext/netshell/lang/pt-PT.rc (hunk 1 above)
Comment (zecarlos1957) [reply dll/shellext/netshell/lang/pt-PT.rc:121]: In WinSrv2003 there is no accelerator in this text.

Grupo

**Comment by zecarlos1957** [comment]
@JoachimHenze 
Do you want me to put the accel?

File: dll/shellext/netshell/lang/pt-PT.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/shellext/netshell/lang/pt-PT.rc:121]: Have you pressed the Alt-key in that dialog? Accels may be configured as being hidden by default.

Comment (binarymaster) [reply dll/shellext/netshell/lang/pt-PT.rc:121]: The `Next >` button has an underline, so I think yes.

Comment (binarymaster) [reply dll/shellext/netshell/lang/pt-PT.rc:121]: Ah no, nevermind, it's just "g".

File: dll/shellext/netshell/lang/pt-PT.rc [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     ReactOS Network Configuration Shell Extension
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR   Copyright 2020-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```
Comment (binarymaster) [inline dll/shellext/netshell/lang/pt-PT.rc:5]:
```suggestion
 * PROJECT:     ReactOS Network Shell
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Portuguese (Portugal) resource file
 * TRANSLATOR:  Copyright 2019-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```

File: dll/shellext/netshell/lang/pt-PT.rc [hunk 3]
```diff
@@ -106,13 +113,13 @@ END
...
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Pretende que este computador seja membro de um domínio?\n\
-(Pode obter esta informação do seu administrador de rede.)", IDC_STATIC, 33, 2, 263, 50
-    AUTORADIOBUTTON "Não, este computador fará parte de um grupo de &trabalho.", IDC_SELECT_WORKGROUP, 33, 32, 253, 20, WS_TABSTOP | WS_GROUP
-    AUTORADIOBUTTON "Sim, este computador fará parte de um &domínio.", IDC_SELECT_DOMAIN, 33, 47, 253, 20, WS_TABSTOP
+    LTEXT "Deseja que este computador seja membro de um domínio?\n\
+(Pode obter esta informação no seu administrador de rede.)", IDC_STATIC, 33, 2, 263, 50
+    AUTORADIOBUTTON "Não; fazer este computador membro do seguinte grupo de trabalho.", IDC_SELECT_WORKGROUP, 33, 32, 253, 20, WS_TABSTOP | WS_GROUP
+    AUTORADIOBUTTON "Sim; fazer este computador membro do seguinte domínio.", IDC_SELECT_DOMAIN, 33, 47, 253, 20, WS_TABSTOP
```
[the remark this answers is not in the document]
Comment (zecarlos1957) [reply dll/shellext/netshell/lang/pt-PT.rc:122]: The semicolon (;) essentially has the same value as the comma, only with the particularity of giving the sentence a longer pause, making the "Yes" or "No" more incisive.

File: dll/shellext/netshell/lang/pt-PT.rc [hunk 4]
```diff
@@ -78,20 +85,20 @@ END
...
+CAPTION "Programa de configuração do ReactOS"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_NET_SETUP, IDC_STATIC, 21, 10, 20, 20
     LTEXT "Seleccione  configuração típica ou personalizada:", IDC_STATIC, 53, 7, 240, 20
     AUTORADIOBUTTON "Configuração &típica", IDC_NETWORK_TYPICAL, 53, 27, 253, 18, WS_TABSTOP | WS_GROUP
     AUTORADIOBUTTON "Configuração &personalizada", IDC_NETWORK_CUSTOM, 53, 72, 253, 18, WS_TABSTOP
-    LTEXT "Cria ligações de rede utilizando Cliente para Redes ReactOS. Ficheiros e Impresoras partilhadas para Redes ReactOS, e o protocolo TCP/IP com endereçamento automático.", IDC_STATIC, 65, 45, 240, 30
-    LTEXT "Permite configurar manualmente componentes de rede.", IDC_STATIC, 65, 90, 243, 20
+    LTEXT "Cria ligações de rede utilizando O ""Cliente para Redes ReactOS"". ""Partilha de ficheiros e de impressão para redes ReactOS"", e o protocolo TCP/IP com endereçamento automático.", IDC_STATIC, 65, 45, 240, 30
```
[the remark this answers is not in the document]
Comment (zecarlos1957) [reply dll/shellext/netshell/lang/pt-PT.rc:95]: No, certainly not. My fault.
Thanks.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**